platforms/__common.sh

```diff
@@ -51,6 +51,75 @@ shouldOverwriteSdk() {
 	fi
 }
 
+shouldOverwritePlatformSdk() {
```

Comment:
nit: looks like GitHub doesn't like displaying tabs vs. spaces in this file 😞 Would you mind cleaning this file up?

Reply:
Thanks for reminding me. I will update it in another PR and avoid rebasing and running builds again.